doc/contributing/contributing_guidelines.md

```diff
@@ -188,7 +189,66 @@ Work in progress.
 [Top](#top)
 
 ##<a name="section9"></a>Documentation
-Work in progress.
+###<a name="section9.1"></a>Repository documentation
+There MUST exist a `doc/` folder at the root of the main repository. Within this folder, there will be a subfolder for each agent type, in addition to a subfolder for `cygnus-common`. For instance:
+
+* `doc/cygnus-ngsi`
+* `doc/cygnus-twitter`
+* `doc/cygnus-common`
+
+All the agent teams MUST provide any documentation aimed to be exposed to the Community within its documentation subfolder.
+
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/949/commits/540563c64137c0bf10bcebaa9cf8f4dfe26093e1